src/state/quiz.js

```diff
@@ -129,17 +130,17 @@ export const quizSavedAction = (value) => {
     });
 
     try {
-      await axios.post(`${apiUrl}/quizzes`, value);
+      const response = await axios.post(`${apiUrl}/quizzes`, value);
 
       dispatch({
         type: "QUIZ_SAVE_SUCCEEDED",
-        data: { title: "", description: "", questions: [] },
+        data: response.data,
       });
     } catch (error) {
       dispatch({
         type: "QUIZ_SAVE_FAILED",
       });
-      console.log(error.message)
+      console.log(error.message);
     }
   };
 };
```

Comment:
in a real world scenario, the ideal thing to do would be put these state changes in a separate PR to keep single purpose PRs. 
no need to change it now, just a reminder ;)

Reply:
I didn't even realize I did this, sorry! On the other PR I was in doubt if I should commit the redux changes together, but since they were related to the changing of themes, I kept it. But I would love to discuss it better